app/views/offers/_form.html.erb

```diff
@@ -49,16 +49,26 @@
     <div class="col">
       <div class="form-group">
         <label for="description">Veröffentlichungsdatum</label>
-        <%=
-          f.date_field :publication_date,
-          class: "form-control",
-          value: f.object.publication_date.try(:to_date) || Time.now.to_date
-        %>
+        <%= f.date_field(
+              :publication_date,
+              :required => true,
+              class: "form-control",
+              value: f.object.publication_date.try(:to_date) || Time.now.to_date
+            ) %>
       </div>
     </div>
   </div>
 
-  <%= render(partial: "shared/partials/dates_form", locals: { record: offer, form: f, fields: ['date_end'], labels: { date_end: "Ablaufdatum" } }) %>
+  <%= render(
+        partial: "shared/partials/dates_form",
+        locals: {
+          record: offer,
+          form: f,
+          fields: ['date_end'],
+          requireds: ['date_end'],
+          labels: { date_end: "Ablaufdatum" },
+          date_end: (Time.now + 4.weeks).to_date
```

Comment:
ich denke es wäre einheitlicher, auch die werte, die im partial vordefiniert werden können, so wie `date_end` hier, als objekt zu übergeben à la `values: { date_end:  (Time.now + 4.weeks).to_date }`, sodass es zu den anderen passt. im partial selbst kann dann geprüft werden, ob es in `values` den gewünschten wert gibt un d dann genommen werden (`.presence`) und es braucht nicht für jeden möglichen wert eine variablendeklaration mit `||= nil` am partialanfang

Reply:
Ist angepasst